ChangeLog

```diff
@@ -7,6 +7,10 @@ Bugfix
      invalidated keys of a lifetime of less than a 1s. Fixes #1968.
    * Fix failure in hmac_drbg in the benchmark sample application, when
      MBEDTLS_THREADING_C is defined. Found by TrinityTonic, #1095
+   * Fix compilation failure for configurations which use compile time
+     replacements of standard calloc/free functions through the macros
+     MBEDTLS_PLATFORM_CALLOC_MACRO and MBEDTLS_PLATFORM_FREE_MACRO.
+     Reported by ole-de and ddhome2006. Fixes #882, #1642 and #1706.
```

Comment:
Minor: #1642 and #1706 is the same issue, reported by same OP. (1642 was raised by myself, after receiving the issue by mail, and then the OP raised 1706). I think we can remove 1642 from the list

Reply:
They're still separate issues on GitHub, and the ChangeLog entry should ideally reflect which Change closed which tickets. So let's leave it like this.

Reply:
I don't mind leaving as is.However, I already closed 1642 as a duplicate